pan3d/dataset_builder.py

```diff
@@ -485,7 +483,6 @@ def export_config(self, config_file=None):
             config["ui"][state_var] = self.state[state_var]
 
         if config_file:
-            with open(config_file, "w") as f:
-                json.dump(config, f)
+            Path(config_file).open("w").write(json.dumps(config))
```

Comment:
https://docs.python.org/3/library/pathlib.html#pathlib.Path.write_bytes

Reply:
Changed in [b816852](https://github.com/Kitware/pan3d/pull/33/commits/b81685249babc1f58e001ca34c9f7d61bc8db8ea)